### ISSUE TITLE
Sybil Attacker Report

### ISSUE THREAD

**Issue by shdwy** [hop-protocol/hop-airdrop#407]
# Related Addresses

```
Chain1:
0x05cc788eff7788add780474ca47b96e0486adf16
0x255a6fdbe0030826aac6b87d97092cfdd4485a11
0x392f10f0b6f59cdcd96c17f4a9d002a39ccfa475
0x81e2e579449aaba9dc68fc1a303454b45115aec0
0x9ddb37e8fd4e865530e850c7fad0279de0040ab0
0xe0b50b58bbf388e17728b1dc6143eef1681d0ce1

Chain2:
0xc128170d8ae71d51dfabdccd1c0ff88444580597
0x11080cc3cb83ae4a4241f48192794cb98c7a75c4
0xc93110d58047bd31f20a302bf2174d15458f6d16
0x59183878fb46b8effedd7fd5e6747bf9df2e55d9
```

# Reasoning

Both chains receive funds (on Polygon) from the previous address on the chain, they interact with the same (or similar) set of contracts and then transfer funds to the next address on the chain. All transactions happened on September 10, 2021.

[Link between chains
](https://polygonscan.com/tx/0xdbd480e408e57ec27b5c66204c96fa7aeb6c2413789677fd600e73019ae7f0a8)

# Methodology

Same as Issue #354 

# Rewards Address

0xD47CB1835EB36e4A108C33E446449192426Fc160

**Comment by shanefontaine** [comment]
Thank you for your submission @shdwy.

Those looks like two independent groups of less than 10 addresses each. Can you provide additional evidence about the connection? The single transaction does not show enough proof.

**Comment by shdwy** [comment]
> Thank you for your submission @shdwy.
> 
> Those looks like two independent groups of less than 10 addresses each. Can you provide additional evidence about the connection? The single transaction does not show enough proof.
Hi! First of all, thank you for your work, I can't imagine processing all this data by myself. All addresses do a 100 Matic swap on Metamask Router on Polygon on Sep 10, 2021 (all in a 1h window).

**Comment by shanefontaine** [comment]
Great find @shdwy. I will update the submission accordingly.

**Comment by shanefontaine** [comment]
Thank you for your report @shdwy. We have verified that the addresses in this report are Sybil attackers.

The report included 10 eligible addresses as Sybil attackers which means you are eligible for 1208.227345955934827457 HOP! When Hop DAO is live, we will make a proposal for this reward — subject to a 1 year lockup, as mentioned in the [original Mirror post](https://hop.mirror.xyz/AI5fOUR0X_l0mktShDOx3mwr-hsB24gp8GvTWtS-MBc).


The qualified addresses are as follows:

```
0x05cc788eff7788add780474ca47b96e0486adf16
0x255a6fdbe0030826aac6b87d97092cfdd4485a11
0x392f10f0b6f59cdcd96c17f4a9d002a39ccfa475
0x81e2e579449aaba9dc68fc1a303454b45115aec0
0x9ddb37e8fd4e865530e850c7fad0279de0040ab0
0xe0b50b58bbf388e17728b1dc6143eef1681d0ce1
0xc128170d8ae71d51dfabdccd1c0ff88444580597
0x11080cc3cb83ae4a4241f48192794cb98c7a75c4
0xc93110d58047bd31f20a302bf2174d15458f6d16
0x59183878fb46b8effedd7fd5e6747bf9df2e55d9
```

**Comment by forweb3** [comment]
sir ，One of my, our report is finding that the rule has to be a jump to survive. However, the successor has completed more than 10 user tracking records, and there is no record of him. I cancelled his reward. And this is what I found so am I not rewarded accordingly? This address has not interacted with hop. And he did not provide other information that he interacted. I report all his reports.
0xD47CB1835EB36e4A108C33E446449192426Fc160 This is a brand new empty address, why can't he provide the address he interacts with, because he is the real empty hunter. He circumvented it with better technical isolation. So here's my reasoning, I want 25% of his reward to go to me and then the remaining 75% to all eligible real users. Just like our rules.
@shanefontaine
